include/dlaf/common/index2d.h

```diff
@@ -294,7 +293,7 @@ LinearIndexT computeLinearIndexRowMajor(const Index2D<IndexT, Tag>& index,
 
   static_assert(std::is_integral<LinearIndexT>::value, "LinearIndexT must be an integral type");
 
-  DLAF_ASSERT_MODERATE(index.isIn(dims), "Index ", index, " is not in the grid ", dims);
+  DLAF_ASSERT_MODERATE(index.isIn(dims), "Index is not in the grid!", dims);
```

Comment:
What about adding also the value of `index`?
```suggestion
  DLAF_ASSERT_MODERATE(index.isIn(dims), "Index is not in the grid!", index, dims);
```